packages/core/src/kernelCoreTemplate.js

```diff
@@ -82,6 +90,43 @@
     Object.freeze(kernel)
     return kernel
 
+    function scuttleGlobalThis(extraAvoids = new Array()) {
+      let props = [...Object.getOwnPropertyNames(globalThis)]
```

Comment:
Why copy the result of getOwnPropertyNames? it's not like something else is using this reference

Reply:
09c4b785d